api/handlers/user.js

```diff
@@ -94,6 +97,49 @@ var UserHandler = {
       }
     },
 
+    share: {
+      get: function(req, res) {
+        if (!req.user) {
+          winston.log('error', 'User not authenticated');
+          return res.status(412).send('Not authenticated');
+        } else {
+          var toHash = req.user.email + process.env.SECRET_KEY;
```

Comment:
I think this should be _id. It's the only field that's always guaranteed to exist, and I also think it's a little strange to put the email in a url